packages/rancher-logging/generated-changes/patch/templates/_helpers.tpl.patch

```diff
@@ -22,3 +22,11 @@
 +  {{- end }}
 +  {{- end }}
 +{{- end }}
++
++{{- define "windowsPathPrefix" }}
++{{- $pathPrefix := (required "Must provide value for .Values.global.cattle.rkeWindowsPathPrefix" .Values.global.cattle.rkeWindowsPathPrefix) -}}
++{{- if (contains "//" $pathPrefix) -}}
++{{- fail ".Values.global.cattle.rkeWindowsPathPrefix must not contain forward-slashes" -}}
```

Comment:
We should handle forward-slashes, we just expect the default to be backslashes

Reply:
This is an old commit. I'll squash.